Issue with peerdependency updating Cypress to V13.1.0

I am using cypress-image-diff-js (V1.28.0) for a project. I am trying to install / update Cypress from V12.17.4 to 13.1. 0 , but this is not possible due to a non resolvable peer dependency used in cypress-image-diff-js for Cypress. 

Could not resolve dependency:
npm ERR! peer cypress@"^9.6.1 || ^10 || ^11 || ^12" from cypress-image-diff-js@1.26.0

--- a/package.json
+++ b/package.json
@@ -66,7 +66,7 @@
     "semantic-release": "^19.0.2"
   },
   "peerDependencies": {
-    "cypress": "^9.6.1 || ^10 || ^11 || ^12"
+    "cypress": ">=9.6.1"
   },
   "resolutions": {
     "y18n": ">=4.0.1",

